client/tailwind.config.js

```diff
@@ -13,14 +12,13 @@ export default {
         800: '#00312F',
         900: '#001E1E'
       },
-      disabled: {
-        500: '#B7B7B7',
-        600: '#E0E0E0'
+      white: {
+        100: 'ffffff',
+        200: 'fefefe'
       },
-      error: {
-        500: '#E76C6C',
-        600: '#C53030'
-      }
+      error: {},
```

Comment:
Curious: Why aren't colors set for these anymore?

Reply:
There was some uncertainty in which shades we will be using, so I've left them blank for now. :)